ccd-definition/CaseEvent/User/UsersEventsDJ-SDO-HNL-nonprod.json

```diff
@@ -4,7 +4,7 @@
   "Name": "Directions Order (Judgment)",
   "Description": "Standard Directions Order (Judgment)",
   "DisplayOrder": 18,
-  "EventEnablingCondition": "detailsOfDirection = \"None\"",
+  "_EventEnablingCondition": "detailsOfDirection = \"None\"",
```

Comment:
This will need to be reverted before merging to master

Reply:
This has been reverted